aml/org.testeditor.aml.dsl/META-INF/MANIFEST.MF

```diff
@@ -9,6 +9,9 @@ Require-Bundle: org.eclipse.xtext,
  org.eclipse.equinox.common;bundle-version="3.7.0",
  org.eclipse.emf.ecore.xcore,
  org.testeditor.aml.model,
+ org.eclipse.osgi,
+ org.eclipse.xtext.generator,

```

Comment:
We should not need to depend on the generator here, should we?
This should go into the `Automated Management of Dependencies` part.